qiita_pet/static/js/networkVue.js

```diff
@@ -2,6 +2,39 @@
 // trigger some functions from other elements
 var processingNetwork = null;
 
+
+/**
+ *
+ * Funtion to format the node labels so they don't overlap
+ *
+ * @param label str The node label
+ *
+ **/
+function formatNodeLabel(label) {
+  // After trying different values, 35 looks like a good value that will not make
+  // the labels overlap in the network.
+  var limit = 35;
+  // Split the input string by the space characters
+  var labelArray = label.split(' ');
+  // For the new label
```

Comment:
For the new label? Not sure what it means ...

Reply:
ok